### PR TITLE
[TASK:P:main] Full fixed french translation

### PR DESCRIPTION
**Pull request by dkd-kaehm**
Fixes: #3983
Ports: #3984 

@megamisan please review this port

### CODE REVIEW

**Comment by dkd-kaehm** [comment]
@megamisan Applied your patch file over my commit.
Please check it again.

File: Resources/Private/Language/fr.locallang.xlf [hunk 1]
```diff
@@ -286,90 +299,98 @@
...
 				<source>The Index Queue manages content indexing. Content enters the Index Queue by initialization below or when new content is created based on configuration. Items in the Index Queue are indexed newest changes first until all items in the queue have been indexed.</source>
-				<target>La file d’attente de l’index gère l’indexation du contenu. Le contenu entre dans la file d&apos;attente de l&apos;index par l’initialisation ci-dessous ou lors de la création d&apos;un nouveau contenu selon la configuration. Les éléments de la file d&apos;attente de l&apos;index sont indexés en commençant par les changements les plus récents jusqu&apos;à ce que tous les éléments de la file d&apos;attente aient été indexés.</target>
+				<target>La file l&apos;indexation gère l&apos;indexation du contenu. Le contenu entre dans la file d&apos;indexation par l&apos;initialisation ci-dessous ou lors de la création d&apos;un nouveau contenu selon la configuration. Les éléments de la file d&apos;indexation sont indexés en commençant par les changements les plus récents jusqu&apos;à ce que tous les éléments de la file d&apos;indexation aient été indexés.</target>
```
Comment (megamisan) [inline Resources/Private/Language/fr.locallang.xlf:308]: Here, I had a typo in my first commit.

```suggestion
				<target>La file d&apos;indexation gère l&apos;indexation du contenu. Le contenu entre dans la file d&apos;indexation par l&apos;initialisation ci-dessous ou lors de la création d&apos;un nouveau contenu selon la configuration. Les éléments de la file d&apos;indexation sont indexés en commençant par les changements les plus récents jusqu&apos;à ce que tous les éléments de la file d&apos;indexation aient été indexés.</target>
```

File: Resources/Private/Language/fr.locallang.xlf [hunk 2]
```diff
@@ -286,90 +299,98 @@
...
 			</trans-unit>
 			<trans-unit id="solr.backend.index_queue_module.flashmessage.success.single_item_was_requeued" xml:space="preserve">
 				<source>Single item was successfully marked for reindexing.</source>
-				<target>Single item was successfully marked for reindexing.</target>
+				<target>L'élément a été marqué pour ré-indexation avec succès.</target>
 			</trans-unit>
 			<trans-unit id="solr.backend.index_queue_module.flashmessage.title" xml:space="preserve">
 				<source>Index Queue</source>
-				<target>File d&apos;attente de l’index</target>
+				<target>File d&apos;&apos;indexation</target>
```
Comment (megamisan) [inline Resources/Private/Language/fr.locallang.xlf:385]: Another one here.

```suggestion
				<target>File d&apos;indexation</target>
```

**Review by megamisan — CHANGES_REQUESTED** [review]
Everything else looks good to me.

**Pull request merged**